hudi-hadoop-common/src/main/java/org/apache/hudi/hadoop/fs/HadoopFSUtils.java

```diff
@@ -109,25 +105,6 @@ public static FileSystem getFs(String pathStr, Configuration conf, boolean local
     return getFs(pathStr, conf);
   }
 
-  public static HoodieStorage getStorageWithWrapperFS(StoragePath path,
```

Comment:
Is this moved to somewhere else?

Reply:
Yeah, since we need to use reflection, I moved it into a new constructor for HoodieHadoopStorage:
```
  public HoodieHadoopStorage(StoragePath path,
                             StorageConfiguration<?> conf,
                             boolean enableRetry,
                             long maxRetryIntervalMs,
                             int maxRetryNumbers,
                             long initialRetryIntervalMs,
                             String retryExceptions,
                             ConsistencyGuard consistencyGuard) {    
    FileSystem fileSystem = getFs(path, conf.unwrapCopyAs(Configuration.class));

    if (enableRetry) {
      fileSystem = new HoodieRetryWrapperFileSystem(fileSystem,
          maxRetryIntervalMs, maxRetryNumbers, initialRetryIntervalMs, retryExceptions);
    }
    checkArgument(!(fileSystem instanceof HoodieWrapperFileSystem),
        "File System not expected to be that of HoodieWrapperFileSystem");
    this.fs = new HoodieWrapperFileSystem(fileSystem, consistencyGuard);
  }
```

Reply:
@yihua to respond later

Reply:
Looks like we have to add a new constructor due to the usage of reflection.  We'll simplify this later.